.github/pull_request_template.md

```diff
@@ -29,17 +29,23 @@ How it Works:
 
 Tests <!-- At least one of them must be included. -->
 
-- Unit test
-- Integration test
-- Manual test (add detailed scripts or steps below)
-- No code
+- [ ] Unit test
+- [ ] Integration test
+- [ ] Manual test (add detailed scripts or steps below)
+- [ ] No code
 
 Side effects
 
-- Performance regression
-    - Consumes more CPU
-    - Consumes more MEM
-- Breaking backward compatibility
+- [ ] Performance regression: Consumers more CPU
+- [ ] Performance regression: Consumers more Memory
+- [ ] Breaking backward compatibility
+
+Documentation
+
+- [ ] Affects user behaviors
+- [ ] Contains syntax changes
+- [ ] Contains variable changes
+- [ ] Contains experimental features
```

Comment:
What do you think about separating out "changes MySQL compatibility"?

This is usually a subset of affects behavior, but specifically with MySQL compatibility the documentation is harder to maintain because it's split between many pages such as MySQL Compatibility, the SQL reference, and sometimes FAQs.

(Affects MySQL compatibility is not always related to syntax: consider clustered index default.)

Reply:
Yes, That would be good.

```suggestion
Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility
```